andela/artemis-ah-backend
# #164198488 setup jwt

#### What does this PR do?
Set up JWT authentication for users

#### Description of Task to be completed?
- Define a helper method to generate a token from a given payload
- Define a helper method to validate a given token and return the subsequent payload

#### How should this be manually tested?
- Clone the repo
- Import util/HelperUtils file
- Call the `HelperUtils.generateToken(payload)` static method to generate a new token.
- Call the `HelperUtils.verifyToken(token)` to verify the validity of a token

#### What are the relevant pivotal tracker stories?
[#164198488](https://pivotaltracker.com/story/show/164198488)
